pytorch/training/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,147 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON=python3
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=3.8.5
+ARG OPEN_MPI_VERSION=4.0.1
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.9
+
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
This arg required to stop docker build waiting for region cofiguration while installing tz data.

Reply:
Please add this as a comment in the dockerfile, so that we know why this was added during future dockerfile changes.